### PR TITLE
migration to new version of clarifai-python-SDK

### PR DESCRIPTION
**Pull request by mogith-pn**
https://clarifai.atlassian.net/browse/TT-2932

### CODE REVIEW

File: pages/compare.py [hunk 1]
```diff
@@ -144,7 +110,7 @@ def list_models(
 pat = load_pat()
 secrets_auth._pat = pat
 secrets_stub = create_stub(secrets_auth)  # installer's stub (PAT)
-
+st.write(f'secrets_stub:{secrets_stub}')
```
Comment (mansi-k) [inline pages/compare.py]: Please remove all these write statements

**Review by mansi-k — CHANGES_REQUESTED** [review]
Please remove all the write() statements used for debugging.... rest looks good

File: pages/compare.py (hunk 1 above)
Comment (mogith-pn) [reply pages/compare.py]: Sure, Mansi !

**Comment by mogith-pn** [comment]
Hi Mansi,
I have cleared the st.write statements, which was used for debugging. Kindly review it and provide your feedback.

**Pull request merged**